src/lib.rs

```diff
@@ -612,14 +586,35 @@ mod tests {
         assert!(vrfy.verify_ecdsa(&msg, &sig, &pk).is_ok());
         assert!(full.verify_ecdsa(&msg, &sig, &pk).is_ok());
 
+        // The following drop will have no effect; in fact, they will trigger a compiler
+        // error because manually dropping a `ManuallyDrop` is almost certainly incorrect.
```

Comment:
Did you mean to write "will trigger a clippy warning"?

Reply:
No, in 1.72 and higher it will be a compiler warning.

Reply:
Oh, my bad.